device-types/Supermicro/SYS-1029U-TR4T.yaml

```diff
@@ -0,0 +1,48 @@
+---
+manufacturer: Supermicro
+model: SuperServer 1029U-TR4T
+slug: supermicro-superserver-1029u-tr4t
+part_number: SYS-1029U-TR4T
+u_height: 1
+is_full_depth: true
+weight: 11.8
+weight_unit: kg
+airflow: front-to-rear
+comments: SuperServer SYS-1029U-TR4T (https://www.supermicro.com/products/system/1U/1029/SYS-1029U-TR4T.cfm)
+console-ports:
+  - name: Serial
+    type: de-9
+    description: Rear port
+power-ports:
+  - name: PSU1
+    type: iec-60320-c14
+    maximum_draw: 750
+  - name: PSU2
+    type: iec-60320-c14
+    maximum_draw: 750
```

Comment:
Hello, thanks for the contribution. It appears that these PSU's are modular. These should be defined under `module-bays` rather than `power-ports`, and then the associated PSU modules should be created within the module-types folder if they do not already exist. Thank you.